omise/views/templates/front/payment-error16.tpl

```diff
@@ -0,0 +1,20 @@
+{capture name=path}
```

Comment:
@jonrandy (optional) To make this file structure consistent with the one in `omise-prestashop/omise/views/templates/hook/` Do you think we should have  
```
- omise/views/templates/front/1.7/payment-error.tpl
- omise/views/templates/front/1.6/payment-error.tpl
```` 
instead?